spec/unit/puppet_pal_2pec.rb

```diff
@@ -71,14 +71,6 @@
         end
         expect(result).to eq(10)
       end
-
-      it '"run_plan" runs a plan with specified content in a manifest' do
```

Comment:
ok to remove together with the impl, but bolt must be in sync and use `call_function('run_plan',...)` in order to continue working.